freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_replica_install.sh

```diff
@@ -5,6 +5,8 @@ set -e
 FQDN=$(hostname -f)
 IPADDR=$(hostname -i)
 
+ipa-server-install -U --uninstall
```

Comment:
nit: replace `-U` with `--unattended` so it will be easier to understand what's going on

Reply:
changed